.github/workflows/cd.yaml

```diff
@@ -7,6 +7,13 @@ on:
     types:
       - completed
 
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+  DOCKER_BUILDKIT: 1
+  GHCR_USERNAME: ${{ github.repository_owner }}
+  GHCR_TOKEN: ${{ secrets.GHCR_TOKEN }}
+
 permissions:
   checks: read
   contents: write
```

Comment:
Are you sure the 403 is not related to missing `packages: write` permission ?

Reply:
No I'm not.